ftests/025-cgset-multiple_cgroups.py

```diff
@@ -23,31 +23,34 @@
 from cgroup import Cgroup, CgroupVersion
 import consts
 import ftests
-import os
 import sys
+import os
 
 CONTROLLER = 'memory'
-CGNAME1 = "025cgset1"
-CGNAME2 = "025gset2"
+CGNAME1 = '025cgset1'
+CGNAME2 = '025gset2'
```

Comment:
Not your bug, but while you're changing this, can you make it `025cgset2`?  Thanks :)

Reply:
Sure, will fix it.